### PR TITLE
New Color's

### PR DESCRIPTION
**Pull request by MicroBotCodes**
This is for BeastFM's new colors.

### CODE REVIEW

File: websites/B/BeastFM/dist/metadata.json [hunk 1]
```diff
@@ -9,14 +9,13 @@
...
   },
   "service": "BeastFM",
-  "version": "1.0.3",
-  "logo": "https://i.imgur.com/l6meesB.png",
-  "thumbnail": "https://i.imgur.com/tWSpYUK.png",
-  "color": "#ff8300",
+  "version": "1.0.4",
+  "logo": "https://i.imgur.com/UtOgutm.png",
+  "thumbnail": "https://i.imgur.com/hLHoPD4.png",
+  "color": "#685884",
```
Comment (Bas950) [inline websites/B/BeastFM/dist/metadata.json:15]: the thumbnail is empty? Use a promotional banner or a picture of the website homepage.

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/B/BeastFM/dist/metadata.json (hunk 1 above)
Comment (MicroBotCodes) [reply websites/B/BeastFM/dist/metadata.json:15]: No, it is the gradient because we don't have a banner.

Comment (Bas950) [reply websites/B/BeastFM/dist/metadata.json:15]: Then take a screenshot of the website homepage...

Comment (MicroBotCodes) [reply websites/B/BeastFM/dist/metadata.json:15]: Yeah it's not updated yet we are switching host and the other one would not update and now the website is down.

Comment (Bas950) [reply websites/B/BeastFM/dist/metadata.json:15]: Alright make a new pr then when you can fix it.

**Pull request closed without merging**